subtests/docker_cli/systemd/systemd_build.py

```diff
@@ -0,0 +1,28 @@
+"""
```

Comment:
Nice breakdown. Well done. But `build` and `pull` are so small, do they really deserve a separate file for each? They could easily fit in `systemd.py`. I agree that `run` is longer and more complicated, so it deserves a separate file. @cevich , thoughts?